src/RemoteRequest/CurlRemoteGetRequest.php

```diff
@@ -146,6 +144,8 @@ static function ($curl, $header) use (&$headers) {
                 continue;
             }
 
+            curl_close($curlHandle);
```

Comment:
In the case of no body being returned, the `continue` would mean the closing wouldn't happen, right?

How about rather just putting `$curlErrno = curl_errno($curlHandle);` right before `curl_close()`?